pom.xml

```diff
@@ -511,7 +511,7 @@
                     </execution>
                 </executions>
                 <configuration>
-                    <forkCount>3</forkCount>
+                    <forkCount>1</forkCount>
                     <reuseForks>false</reuseForks>
                     <skipTests>${skipTests}</skipTests>
                     <skipITs>${skipITs}</skipITs>
```

Comment:
This file should not be changed, i will revert it back once eiffel commons is merged, i will need to use 0.0.15 only.